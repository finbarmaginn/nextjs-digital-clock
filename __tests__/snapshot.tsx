import { render } from '@testing-library/react'
import Home from '@/pages/index'
import About from '@/pages/about'

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})

it("renders about page unchanged", () => {
  const { container } = render(<About />);
  expect(container).toMatchSnapshot();
});
