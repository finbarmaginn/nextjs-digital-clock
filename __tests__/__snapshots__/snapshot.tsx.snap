// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <div>
        <h1
          role="heading"
        >
          Logo
        </h1>
      </div>
      <div>
        <nav
          class="nav"
        >
          <ul>
            <li>
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                href="/about"
              >
                About
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <main>
      <h2
        role="heading"
      >
        About
      </h2>
      <article>
        <section>
          <p>
            This web application is built using NextJS, React, Jest, Modular CSS etc etc
          </p>
        </section>
      </article>
    </main>
    <footer>
      Developed by Finbar Maginn
    </footer>
  </div>
</div>
`;

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <div>
        <h1
          role="heading"
        >
          Logo
        </h1>
      </div>
      <div>
        <nav
          class="nav"
        >
          <ul>
            <li>
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                href="/about"
              >
                About
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <main>
      <h2
        role="heading"
      >
        Home
      </h2>
    </main>
    <footer>
      Developed by Finbar Maginn
    </footer>
  </div>
</div>
`;
